source/_posts/2015-03-26-creating-a-dynamic-single-page-app-for-our-genome-team-using-react.markdown

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: "Creating a dynamic single page app for our Genome Team using React"
+date: 2015-03-26 10:42
+comments: true
+categories: [React]
```

Comment:
There are some other categories you might include [here](http://artsy.github.io/blog/2014/09/05/we-open-sourced-our-isomorphic-javascript-website/).